xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/edit_confirmation/lock.vm

```diff
@@ -0,0 +1,27 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+##
+## Since 15.9RC1
+## The template used by XWikiDocumentLockEditConfirmationChecker, used in case of page locked by another user.
+##
+#set ($lockingUser = "#displayUser($tdoc.lockingUser {'useInlineHTML': true})")
+<p class="xwikimessage">
+  $escapetool.xml($services.localization.render('doclockedby')) ${lockingUser.trim()}.
```

Comment:
Isn't $lockingUser a parameter of the translation?

Reply:
It's not, but indeed could be interesting to introduce a better translation with params during the refactoring.

Reply:
I've introduced `platform.core.editConfirmation.lock.lockedBy` with a param. Note that `doclockedby` is still used in several placed. I haven't updated as some of the required to be careful with parameters escaping.